samples/system-test/detect.test.js

```diff
@@ -17,12 +17,12 @@
 
 const path = require('path');
 const {assert} = require('chai');
-const execa = require('execa');
-
+const cp = require('child_process');
 const cmd = 'node detect.js';
 const cmd_tts = 'node detect-intent-TTS-response.v2.js';
 const cmd_sentiment = 'node detect-intent-sentiment.v2.js';
 const cwd = path.join(__dirname, '..');
+cp.cwd = cwd;
```

Comment:
We can just delete lines 24 and 25 here

Reply:
Also - we like to add a line like this:

```js
const exec = cmd => execSync(cmd, { encoding: 'utf8' });
```

We use this pretty consistently to make sure the data returned from `execSync` is properly encoded.  We also generally don't do the `pipe` bit below